desktop/apps/armory_week/index.js

```diff
@@ -4,8 +4,11 @@ import { renderLayout } from '@artsy/stitch'
 import adminOnly from '../../lib/admin_only'
 import JSONPage from '../../components/json_page/es6'
 import ArmoryWeekPage from './components/ArmoryWeekPage'
+import merge from 'lodash.merge'
+import queryString from 'query-string'
 
 const SLUG = 'armory-week'
+const MARKETING_MODAL_ID = 'ca18'
```

Comment:
Does this value ever change? We could avoid the hard code by making it an env var and access via Sharify but I don't know how relevant that will be.

Reply:
We could change a slug name if we really wanted to, but we usually don't do that once we assign a slug name to a campaign. I have already created and reserved `ca18` both in [staging](https://staging.artsy.net/marketing-signup-modals) and [production](https://www.artsy.net/marketing-signup-modals) and it shouldn't be an issue unless there's a strong argument against using the name `ca18` specifically.